### PR TITLE
Bump cardano-ledger-specs and fix breaking change

### PR DESCRIPTION
**Pull request by kk-hainq**

### CODE REVIEW

File: cabal.project [hunk 1]
```diff
@@ -211,7 +211,7 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-ledger-specs
-  tag: 51561fcb13170e3849873eef82e634c1041ee1d6
+  tag: 30eca73a2f5c13f1fbed9a98a59540ac3d0c8afe
   --sha256: 1699hbz1i92hbjjzv9ym43qzji5xiz7gqbd0y8m2ylc353c0sxzn
```
Comment (nc6) [inline cabal.project:215]:
```suggestion
  --sha256: 02klj4zpcl98p22n2v9nkjxxdf8mmw069mw4ddhlgr931bfigz6w
```

**Review by nc6 — CHANGES_REQUESTED** [review]
You'll need to fix the sha hash for the ledger repo. I've suggested the fix.

**Comment by kk-hainq** [comment]
> You'll need to fix the sha hash for the ledger repo. I've suggested the fix.

Sorry I was careless seeing the line commented. Have fixed as suggested!

**Comment by nfrisby** [comment]
Hi @kk-hainq , thanks for fixing this for us.

Please squash your two commits into one before merging this PR. Otherwise, LGTM. Thanks.

**Comment by kk-hainq** [comment]
> Hi @kk-hainq , thanks for fixing this for us.
> 
> Please squash your two commits into one before merging this PR. Otherwise, LGTM. Thanks.

I have, thanks!

**Review by nfrisby — APPROVED** [review]
Thanks much. Approving. And I'll merge too.

**Comment by nfrisby** [comment]
Hi @kk-hainq. Thanks again for the helpful commit.

I was a bit lax this time, since this commit is so simple. But generally -- and usually even for a small one -- we'd like to have a few more details in the commit message and a non-empty PR description. (The description can even just say "Please see the commit messages" eg).

Thanks again!

**Comment by kk-hainq** [comment]
> Hi @kk-hainq. Thanks again for the helpful commit.
> 
> I was a bit lax this time, since this commit is so simple. But generally -- and usually even for a small one -- we'd like to have a few more details in the commit message and a non-empty PR description. (The description can even just say "Please see the commit messages" eg).
> 
> Thanks again!

Yes sir. I should have at least added a reference to the breaking commit in ledger. Will add more information next time!

**Pull request merged**